forge_sync/__init__.py

```diff
@@ -0,0 +1,11 @@
+from dagster import Definitions, load_assets_from_modules
+
+from . import assets
+from .resources import MongoDBResource
+
+all_assets = load_assets_from_modules([assets])
+
+defs = Definitions(
+    assets=all_assets,
+    resources={"db_conn": MongoDBResource(database="forge_sync", collection="finance_metrics_low")},
```

Comment:
```suggestion
    resources={"db_conn": MongoDBResource(database="forge_sync", collection="finance_metrics_daily")},
```
trocar o nome para um com sentido embutido da granularidade da data tipo diário